### DISCUSSION TITLE
how to swap between screen ?

### DISCUSSION THREAD

**Discussion by c-benoit** [deskflow/deskflow#7874] [Q&A] [answered]
Hello, i connected my two computers
But impossible to move the mouse between screen
Started looking for edge crossing / switching option but without success
How to cross by default, and how to cross without pressing key (seemlessly)

I'm using a w11 as main computer, and macos as secondary
*update > I can teleport my mouse using a Hotkey (keystroke F8 - switchInDirection(right)), but i really want to swap between computers without pressing keys

thanks in advance!

**Comment by sithlord48** [comment]
Well this is literally impossible to answer, You have given us nothing to help you with.

Can you tell us

What OS(es) your using 

What version of deskflow your using (or versions) ?

If your using linux are you using wayland , and what desktop your using ?

What settings in deskflow have you changed on the client/ server (if any)?

What have you tried ?

**Comment by c-benoit** [comment, marked as answer]
Hello, nvm 🙏 >> i updated the post with the informations

**Comment by sithlord48** [comment]
Do you lock to screen enabled ? Scroll Lock is the default key for that

**Reply by c-benoit** [reply]
i tried with and without without success
![image](https://github.com/user-attachments/assets/d909bdf3-28ae-4601-bdcc-a9c50a162857)

**Comment by c-benoit** [comment]
hello, the problem is still present unfortunately
